### ISSUE TITLE
Dashboard: Too few Dates Displayed on Axis

### ISSUE THREAD

**Issue by mattholy** [metabase/metabase#12699]
**Describe the bug**
As you can see on the screenshots below, there are only 2 dates that are displayed on the axis where it might can be placed 5 dates at least.
* resolution: 1920*1080

**Screenshots**
![Snipaste_2020-06-15_15-50-33](https://user-images.githubusercontent.com/22177897/84631466-00798680-af20-11ea-98da-5be6682de206.png)

:arrow_down: Please click the :+1: reaction instead of leaving a `+1` or `update?` comment

**Comment by flamber** [comment]
Looks similar to #11158, but not specific to months. Related to #12684
Steps to reproduce:
1. Simple question > Sample Dataset > Orders
2. Filter CreatedAt between 2020-04-01 and 2020-04-30
3. Summarize Count by CreatedAt:Day and Category
4. Visualization Settings > Data > CreatedAt :gear: > Style: yyyy/m/d
![image](https://user-images.githubusercontent.com/1447303/84644334-525de480-aeff-11ea-92b8-8540018d4831.png)
On dashboard, where cards are 9, 4, 13, 14, 18 columns
![image](https://user-images.githubusercontent.com/1447303/84645906-9c47ca00-af01-11ea-8d55-7cf39e3674c0.png)
5. Style: mmmm d, yyyy
![image](https://user-images.githubusercontent.com/1447303/84644781-e9c33780-aeff-11ea-99e6-212bad2f6f95.png)
On dashboard, where cards are 9, 4, 13, 14, 18 columns
![image](https://user-images.githubusercontent.com/1447303/84646019-c600f100-af01-11ea-9612-fd39e043403c.png)

**Comment by lorem--ipsum** [comment]
This issue is four and a half years old, we've moved to a different chart library since then. @alxnddr should we close this?

**Comment by cdeweyx** [comment]
Agreed this has gotten better and is hard to repro in v53 — Going to close for now and let's open more specific issues if this surfaces again for specific charts / cases.
